templates/support.rackspace.com/home.html

```diff
@@ -1,5 +1,12 @@
 {% extends "_layouts/base.html" %}
 
+
+## ALEX DOESNT KNOW WHAT SHE IS DOING HERE
```

Comment:
We're actually going to have to fix this in a completely different repo; sorry about that as I completely forgot. Here's where to go: https://github.com/rackerlabs/docs-support-header-as-service/blob/master/src/html/header.html#L67 (if you don't have access, let me know). We'll have to see if we can just remove the form and replace with this, or if we need to weave it in.